packages/venia-ui/lib/components/CartPage/ProductListing/productListing.js

```diff
@@ -28,41 +29,13 @@ const ProductListing = props => {
     }
 };
 
-export default ProductListing;
-
-export const CartBody = gql`
-    fragment CartBody on Cart {
-        id
-        items {
-            id
-            product {
-                name
-                small_image {
-                    url
-                }
-            }
-            prices {
-                price {
-                    currency
-                    value
-                }
-            }
-            quantity
-            ... on ConfigurableCartItem {
-                configurable_options {
-                    option_label
-                    value_label
-                }
-            }
-        }
-    }
-`;
-
 export const GET_PRODUCT_LISTING = gql`
     query getProductListing($cartId: String!) {
         cart(cart_id: $cartId) {
-            ...CartBody
+            ...ProductListingFragment
         }
-        ${CartBody}
+        ${ProductListingFragment}
```

Comment:
Odd that this works, but the fragment should be outside of the query body.

```
query ...
${ProductListingFragment}
```

Reply:
Oops